examples/operator/101_initial_cluster.yaml

```diff
@@ -9,13 +9,13 @@ metadata:
 spec:
   images:
     vtctld: vitess/lite:v16.0.0-rc1
-    vtadmin: vitess/vtadmin:latest
+    vtadmin: vitess/vtadmin:v16.0.0-rc1
```

Comment:
Are we going to change all the images to the GA version of v16, once v16 GA is released?

Reply:
Thank you @GuptaManan100